src/utils/formatDate.js

```diff
@@ -0,0 +1,13 @@
+export function formatDate(date) {
+  const dateFormatOptions = {
+    year: "2-digit",
+    month: "2-digit",
+    day: "2-digit",
+  };
```

Comment:
consider introducing a variable and reuse where applicable.

```suggestion
  const format = '2-digit'
  const dateFormatOptions = {
    year: format,
    month: "2-digit",
    day: "2-digit",
  };
```